src/constants/fake-data/leaderboard.ts

```diff
@@ -6,284 +6,222 @@ export const leaderboardData: LeaderboardData[] = [
   {
     rank: 1,
     score: 10000,
-    address: "1BoatSLRHtKNngkdXEeobR76b53LETtpyT",
-    avatar: "🙂",
     username: "AliceInWonderland",
-    gamesPlayed: 50,
-    payout: 30,
+    matchesPlayed: ["a", "b"],
```

Comment:
Should this string comply with the UUID type?

Reply:
This file will be deleted so it doesn't matter